internal/exp/metrics/identity/resource.go

```diff
@@ -0,0 +1,28 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package identity
+
+import (
+	"hash"
+	"hash/fnv"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatautil"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+)
+
+type Resource struct {
+	attrs [16]byte
```

Comment:
Do we need to include the resource SchemaURL as part of the identifier?

Reply:
no idea, can someone clarify? @jpkrohling @djaglowski

Reply:
Yes: https://opentelemetry.io/docs/specs/otel/resource/sdk/#create

Reply:
This came up in the maintainers meeting yesterday and consensus seemed to be that most language libraries are not setting this field. Technically it is optional so I would be ok with leaving it out for now and adding later if necessary.